huaweicloud/services/acceptance/lts/resource_huaweicloud_lts_cce_access_test.go

```diff
@@ -0,0 +1,526 @@
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getCceAccessConfigResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckLTSCCEAccess(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testCceAccessConfigHostFile(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttrSet(rName, "log_group_name"),
+					resource.TestCheckResourceAttrSet(rName, "log_stream_name"),
+					resource.TestCheckResourceAttr(rName, "tags.key", "value"),
+					resource.TestCheckResourceAttr(rName, "tags.foo", "bar"),
+					resource.TestCheckResourceAttr(rName, "access_type", "K8S_CCE"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.paths.0", "/var"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.black_paths.0", "/var/a.log"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.path_type", "host_file"),
+				),
+			},
+			{
+				Config: testCceAccessConfigHostFileUpdate(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "tags.key", "value-updated"),
+					resource.TestCheckResourceAttr(rName, "tags.owner", "terraform"),
+					resource.TestCheckResourceAttr(rName, "access_type", "K8S_CCE"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.paths.0", "/var/logs"),
+					resource.TestCheckResourceAttr(rName, "access_config.0.black_paths.0", "/var/logs/a.log"),
+				),
+			},
+			{
+				ResourceName:      rName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccCceAccessConfigImportStateFunc(rName),
+			},
+		},
+	})
+}
+
+func testAccCceAccessConfigImportStateFunc(rName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[rName]
+		if !ok {
+			return "", fmt.Errorf("resource (%s) not found: %s", rName, rs)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		if name == "" {
+			return "", fmt.Errorf("can not find the 'name' parameter in %s", rName)
+		}
+		return name, nil
+	}
+}
+
+func testCceAccessConfigContainerFile(name, updateStr string) string {
+	return fmt.Sprintf(`
+resource "huaweicloud_lts_cce_access" "container_file" {
+  name           = "%s"
+  log_group_id   = "%s"
+  log_stream_id  = "%s"
+  host_group_ids = ["%s"]
+  cluster_id     = "%s"
+
+  access_config {
+    path_type            = "container_file"
+    paths                = ["/var"]
+    black_paths          = ["/var/a.log"]
+    name_space_regex     = "namespace%[6]s"
+    pod_name_regex       = "podname%[6]s"
+    container_name_regex = "containername%[6]s"
```

Comment:
which format do you prefer？with index or without

Reply:
done